internal/cmd/commands/authenticate/oidc.go

```diff
@@ -103,87 +106,49 @@ func (c *OidcCommand) Run(args []string) int {
 		return base.CommandCliError
```

Comment:
Not directly relevant to this PR but in the case that the user is trying to authenticate where a browser isn't available on the same platform could we print out the URL and allow them to open it manually?

Reply:
Good point. If we get an error back from the open I think we should print it out. I'll fix that.